sdk/resourcemanager/autocent.js

```diff
@@ -13,7 +13,9 @@ const pkgRegEx2 = /Package\s+tag\s+(.+)\.</;
 const data = {};
 const servicesInvalidUrl = ["securitydevops"];
 const deprecatedArtifacts = [
-    "azure-resourcemanager-loadtestservice"
+    "azure-resourcemanager-loadtestservice",
```

Comment:
Why this line changes? Seems it's modified in last PR?
https://github.com/Azure/azure-sdk-for-java/pull/44221/files#diff-8db1cd8c365b4b4809d9564632acc6785e63dc0a1dea5d58f690e86c715d2419R16

Reply:
Fixed by [35aba77](https://github.com/Azure/azure-sdk-for-java/pull/44222/commits/35aba7760936fc5f9ac40c5ade74274a537fd54c)

Reply:
What's the difference from the already merged PR?
https://github.com/Azure/azure-sdk-for-java/pull/44221/files

Reply:
release branch?

Reply:
I see.

Reply:
@weidongxu-microsoft 
Yes, this PR is for release branch.